libraries/core-react/src/components/Banner/Banner.tokens.ts

```diff
@@ -15,32 +16,63 @@ const {
       primary__moss_green_100: { rgba: infoColor },
       primary__energy_red_100: { rgba: warningColor },
     },
+    ui: {
+      background__default: { rgba: backgroundColor },
+    },
   },
   shape: {
     circle: { minHeight, minWidth, borderRadius },
   },
 } = tokens
 
-export const banner = {
-  enabled: {
-    typography: {
-      ...messageTypography,
+export type BannerToken = ComponentToken & {
+  entities?: {
```

Comment:
Probably dont need this optional flag?

Reply:
I know, that was the one thing I noticed in VS Code after the commit, hoped no one would notice 😂

Reply:
ftfy